node-backend/subProblemHandler.js

```diff
@@ -116,12 +116,12 @@ module.exports = function(db) {
   function deleteSubproblem(request, response, next) {
     const subproblemId = request.params.subproblemId;
     const workspaceId = request.params.workspaceId;
-    logger.debug('Start deleting subproblem ' + subproblemId);
+    logger.debug('Start deleting subproblem: ' + subproblemId);
     db.runInTransaction(_.partial(deleteTransaction, workspaceId, subproblemId, next), function(error) {
       if (error) {
         util.handleError(error, next);
       } else {
-        logger.debug('done deleting subProblem : ' + JSON.stringify(subproblemId));
+        logger.debug('Done deleting subproblem: ' + JSON.stringify(subproblemId));
```

Comment:
again with the stringify, v. weird. the `+` operator already handles this, surely?